gneiss/balances.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import division
+import numpy as np
+from skbio.stats.composition import clr_inv
+from collections import OrderedDict
+
+
+def _balance_basis(tree_node):
```

Comment:
You're suggesting to move this all code block into `_count_matrix` correct?

We could do that.  The main reason why I wanted to keep them separate was because I wanted to be able to test these modules separately.  What do you think about that?